src/plugins/vis_types/timeseries/public/application/components/aggs/serial_diff.js

```diff
@@ -85,24 +89,33 @@ export const SerialDiffAgg = (props) => {
           <EuiFormRow
             id={htmlId('lag')}
             label={
-              <FormattedMessage
-                id="visTypeTimeseries.serialDiff.lagLabel"
-                defaultMessage="Lag"
-                description="'Lag' refers to the parameter name of the serial diff translation
+              <>
+                <FormattedMessage
+                  id="visTypeTimeseries.serialDiff.lagLabel"
+                  defaultMessage="Lag"
+                  description="'Lag' refers to the parameter name of the serial diff translation
                 https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-serialdiff-aggregation.html.
                 This should only be translated if there is a reasaonable word explaining what that parameter does."
-              />
+                />{' '}
+                <EuiIconTip
```

Comment:
You added `IconTip`, pleas reflect it in the description and add a screen

Reply:
It had already added in description.